ccmlib/__init__.py

```diff
@@ -0,0 +1,6 @@
+import logging
+import sys
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
```

Comment:
Those should go into ccm script.

If ccm is used as a library it should be configuring the logging output